src/components/Home.jsx

```diff
@@ -8,11 +8,14 @@ import { ZooniverseLogotype, ZooniverseLogo } from 'zooniverse-react-components'
 import { config, subjectSets } from '../config';
 import SocialSection from '../components/SocialSection';
 import { selectSubjectSet } from '../ducks/subject';
+import { setGoldStandard } from '../ducks/workflow';
 import Divider from '../images/img_divider.png';
 import BostonLogo from '../images/BPL_logo.jpg';
 import IMLSLogo from '../images/imls_logo.png';
 
 const BUFFER = 50;
+const EXPERT = 'expert';
```

Comment:
[minor] Since "user role" is a sorta-enumerated value, it might make sense to conceptually organise this into `const USER_RULES = { EXPERT: 'expert', OWNER: 'owner' }`

[minor] Would we also need a USER_ROLES.GENERIC type? I need to check the rest of the code.

This would be matters for future improvement, though.